recipes-support/blink-led/blink-led/blink-led

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+led="red"
+
+if [ "$led" = "red" ]; then
+
+echo 0 > /sys/class/leds/led-green/brightness
+echo 0 > /sys/class/leds/led-blue/brightness
+
+echo heartbeat > /sys/class/leds/led-red/trigger
```

Comment:
Remove one if-statement including its inner scope completely and also remove the outer if-statement of the other one. Then parse the used color from your recipe using `sed` directly. For this, use a "variable" in the shell script, something like `@COLOR@`. In the end, you should have left only the three `echo`s setting the LED brightness and trigger.